instrumentation/opentelemetry/metrics.go

```diff
@@ -44,6 +46,8 @@ func (mh *HttpOperationMetricsHandler) CreateRequestCount() {
 		otel.Handle(err)
 	}
 
+	mh.countersMutex.Lock()
+	defer mh.countersMutex.Unlock()
```

Comment:
So we think `AddToRequestCount` is racing with `CreateRequestCount`? But if I look at the code it looks like `CreateRequestCount` would have had to happen first as part init of `WrapHandler`? 

Also we don't really need a map here right? Because we have a static set of metrics and can precreate counters?

Reply:
This made me think about how this is designed and I agree with you. I need to maybe get rid of the map and switch to static counters. I had a map because I thought there would be more counters we would add but we do not need a map if we can reference the counters directly.